extensions/warp-mp-ipfs/src/store/mod.rs

```diff
@@ -26,6 +26,10 @@ pub const IDENTITY_BROADCAST: &str = "identity/broadcast";
 pub const FRIENDS_BROADCAST: &str = "friends/broadcast";
 pub const SYNC_BROADCAST: &str = "/identity/sync/broadcast";
 
+pub fn get_inbox_topic(did: &DID) -> String {
```

Comment:
this probably should go into the friends.rs file

Reply:
ok, we can think of a generalization of that concept in the future